Add information about API Elements and subpackages to the README (#57)
apiaryio/api-elements.js: README.md

# API Elements: JavaScript

[API Elements](https://apielements.org/) is a uniform interface for dealing

> with API Descriptions. This repository contains tooling for handling API

Reviewer: ```suggestion
with API description formats ([API Blueprint](https://apiblueprint.org/), [OpenAPI](https://github.com/OAI/OpenAPI-Specification), ...). This repository contains tooling for handling API
```

> Elements in JavaScript along with parsers and serializers for API Description

Reviewer: ```suggestion
Elements in JavaScript along with parsers and serializers for API description
```

Reviewer: Hmm, API description languages mean the same as API description formats. I wouldn't argue about domain dictionary now, but I suggested formats above and we should be at least consistent.

languages.

> API Element Adapters:

Reviewer: ```suggestion
API Elements adapters:
```

- [API Blueprint Parser](packages/fury-adapter-apib-parser)
- [API Blueprint Serializer](packages/fury-adapter-apib-parser)
- [OpenAPI 2 Parser](packages/fury-adapter-swagger)

## Usage

```javascript
const { Fury } = require('fury');

const fury = new Fury();

> // Load any parsers or serializer adapters you wise to use

Reviewer: wise?

Author: wish*

const apiBlueprintParser = require('fury-adapter-apib-parser');
fury.use(apiBlueprintParser);

const openAPI2Parser = require('fury-adapter-swagger');
fury.use(openAPI2Parser);

> const source = 'FORMAT: 1A\n# My API');

Reviewer: Perhaps a multiline string which looks like _API description_ would do a better justice to the example:

```js
const source = `
FORMAT: 1A

# My API
## GET /message
+ Response 200 (text/plain)

        Hello World!
`;
```

(the document stolen from the APIB homepage)

fury.parse({source}, (error, parseResult) => {
  console.log(parseResult.api.title);
});
```

See [API Reference documentation](https://api-elements-js.readthedocs.io/en/latest/api.html#elements)

> for the details about ParseResult object and other elements interface in

Reviewer: ```suggestion
for the details about the ParseResult object and other elements interface in
```
